stacks/bottlerocket-ecs-updater.yaml

```
@@ -119,6 +130,7 @@ Resources:
           LogConfiguration:
             LogDriver: awslogs
             Options:
+              awslogs-create-group: 'True'
```

nit: [`strconv.ParseBool` accepts both](https://golang.org/pkg/strconv/#ParseBool) but the [`awslogs` driver documentation specifies "true"](https://docs.docker.com/config/containers/logging/awslogs/#awslogs-create-group)
```suggestion
              awslogs-create-group: 'true'
```

---

changed and re-tested.